### PR TITLE
Workflow improvement - add option for non-PR manual builds

### PR DESCRIPTION
**Pull request by nerdCopter**
* allow manually triggered builds via `manual-build.yml` (artifacts only, no release).
* allows fork-owners to easily build all OS versions via actions tab.
* useful for fork-owner testing, prior to making PR's.
* names the workflow-run appropriately with branch-name and `debug` if applicable.
* reference discussions in https://github.com/betaflight/betaflight-configurator/pull/3043

image

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Review by blckmn — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Pull request merged**
